app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list | Player) -> int:
+    total = 0
+    for player in players:
+        total += player.get_rating()
+    return total
```

Comment:
if players will be type of Player - you will get exception that Player is not iterable. It should be better list[Player]

Reply:
Same for other functions